src/us/kbase/workspace/database/provenance/ResolvablePID.java

```diff
@@ -0,0 +1,118 @@
+package us.kbase.workspace.database.provenance;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * A resolvable permanent identifier for a resource.
+ */
+public class ResolvablePID {
+
+	private final String id;
+	private final String description;
```

Comment:
Is this a description of the ID or the referent of the ID? The docs say it's a description of the ID but that doesn't seem particularly helpful

Reply:
Ah, I will clarify that. Thanks!

Reply:
:+1: